README.md

```diff
@@ -16,6 +16,92 @@
 
 </div>
 
+## Getting Started
+
+To use the Svelte Notifications library in your Svelte application, you can install it using npm:
+
+```bash
+npm install @whizzes/svelte-notifications
+```
+
+## Usage
+
+Then, you can import the NotificationList and the notification into your component and start using it.
+
+```javascript
+import { NotificationList, notifications } from '@whizzes/svelte-notifications';
+```
+
+### Provide the NotificationList component
+
+The NotificationList component is the container of the notifications. It should be placed at the layout of your application, so it can be visible in all the pages.
+
+You can set the position of the notifications by passing the `position` prop to the NotificationList component. The available positions are the following using the `Position` enum:
+
+- `Position.TopLeft`
+- `Position.TopCenter`
+- `Position.TopRight`
+- `Position.BottomLeft`
+
+```html
+<script>
+  import { NotificationList, notifications, Position } from '@whizzes/svelte-notifications';
+</script>
+
+<NotificationList position={Position.BottomRight} let:notification>
+  <!-- Your notifications template -->
+  <li>
+    <strong>{notification.title}</strong>
+    <p>{notification.message}</p>
+  </li>
+</NotificationList>
+
+```
+
+### Add a notification
+
+To add a notification, you can use some of the methods available in the `notifications` object.
+
+```javascript
+import { notifications } from '@whizzes/svelte-notifications';
+
+// Add a success notification
+notifications.notifySuccess('Hello World!');
+// Add an error notification
+notifications.notifyError('Hello World!');
+// Add a warning notification
+notifications.notifyWarning('Hello World!');
+// Add an info notification
+notifications.notifyInfo('Hello World!');
+```
+
+arguments:
+
+0. message: string
+1. title?: string
+
+### Example
+
+```html
+<script lang="ts">
+  import { NotificationList, notifications, Position } from '$lib';
+
+  const success = () => {
+    notifications.notifySuccess('Hello World!');
+  };
+</script>
+
+<button on:click={success}>Append Success</button>
+
+<!-- use position prop to change the position of the notification list -->
+<NotificationList position={Position.BottomRight} let:notification>
+  <li>
+    <strong>{notification.title}</strong>
+    <p>{notification.message}</p>
+  </li>
+</NotificationList>
+```
```

Comment:
This is a nice example to move at the very top, because it shows you how the library is used in a single picture